setup.sh

```diff
@@ -1,28 +1,31 @@
 #!/bin/bash -e
 
 #####################################################
-#
 # Set up the Membership application
-#
 #####################################################
 
-echo " ____ ____ ____ ____ ____ ____ ____ ____ ____ ____ ";
-echo "||m |||e |||m |||b |||e |||r |||s |||h |||i |||p ||";
-echo "||__|||__|||__|||__|||__|||__|||__|||__|||__|||__||";
-echo "|/__\|/__\|/__\|/__\|/__\|/__\|/__\|/__\|/__\|/__\|";
-
-printf "\n\r"
-
-echo "  ^    ^    ^    ^    ^    ^    ^    ^  "
-echo " /f\  /r\  /o\  /n\  /t\  /e\  /n\  /d\ "
-echo "<___><___><___><___><___><___><___><___>"
-
```

Comment:
I let @rtyley remove [this](https://github.com/guardian/membership-frontend/commit/1cd84b09b9fda3a71260fa4989448f5f89862fe3#diff-adf92bc2bafe2cc674ca55ab2dfd13c3L10) and have never forgiven myself since. Not on my watch.